float64 x_c
float64 y_c
float64 psi
float64 dx_c
float64 dy_c
float64 dx_c_v
float64 dy_c_v
float64 ax_c_v
float64 ay_c_v
float64 dpsi
float64 del_s_act
float64 omega_wheel_fl
float64 omega_wheel_fr
float64 omega_wheel_rl
float64 omega_wheel_rr
float64 t_mot_fl_act
float64 t_mot_fr_act
float64 t_mot_rl_act
float64 t_mot_rr_act
float64 dt_mot_fl_act
float64 dt_mot_fr_act
float64 dt_mot_rl_act
float64 dt_mot_rr_act
float64 fx_fl
float64 fy_fl
float64 fx_fr
float64 fy_fr
float64 fx_rl
float64 fy_rl
float64 fx_rr
float64 fy_rr
float64 del_s_ref
float64 t_mot_fl_ref
float64 t_mot_fr_ref
float64 t_mot_rl_ref
float64 t_mot_rr_ref
